coverity/scan.groovy

```diff
@@ -0,0 +1,31 @@
+pipeline {
+    agent {
+        docker {
+            image 'plavjanik/coverity-oss'
```

Comment:
Would it make sense to move to `zowe` project instead of your name?

Reply:
I will bring it to Mark Ackert. I cannot do it myself.